handlers/bootstrap.go

```diff
@@ -60,6 +62,8 @@ func ActivateMachine(event *events.Event, apiClient *client.RancherClient) error
 		return err
 	}
 
+	publishTransitioningReply("Creating agent container...", event, apiClient)
```

Comment:
The pullImage call is going to be the longest pole in this tent. I _think_ we have the ability to get the progress of the image pull, but I haven't tried it.

Do you think seeing the progress of them image pull would be useful to the user? It would fill in an otherwise big silent gap in the progress.

I think we can do this by specifying the OutputStream on the PullImageOptions. Take a look at the code: https://github.com/fsouza/go-dockerclient/blob/master/image.go#L258-L270 if you think this is worth pursing.

Let me know if you want to discuss it.